addons/mail/static/src/core/public_web/thread_model_patch.js

```diff
@@ -17,7 +17,7 @@ patch(Thread.prototype, {
         if (["group", "chat"].includes(this.channel_type)) {
             return this.store.discuss.chats;
         }
-        if (this.channel_type === "channel") {
+        if (this.channel_type === "channel" && !this.parent_channel_id) {
```

Comment:
This is preventing them in sidebar but nothing is done for messaging menu, is that on purpose?

subchannels are displayed as their own in messaging menu, I guess it's an ok oversight for now, but we might want to change somethingThis is preventing subchannels in sidebar but nothing is done for messaging menu, is that on purpose?

Subchannels are displayed as their own in messaging menu, I guess it's an ok oversight for now, but we might want to change something eventually.

Reply:
Yeah, the same holds for the command palette. We should improve sub-channels displaying later on. By clearly indicating they are sub-channels. But I think they deserve to be shown in messaging menu/command palette.